.github/workflows/preview.yml

```
@@ -0,0 +1,53 @@
+        with:
+          node-version: "20"
+          cache: 'yarn'
+          
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+        
+      - name: Build
+        run: yarn build:prod
+        
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)